apps/condo/domains/meter/hooks/useUpdateMeterModal.tsx

```diff
@@ -45,10 +45,16 @@ export const useUpdateMeterModal = (refetch) => {
     }, [selectedMeter])
 
     const handleSubmit = useCallback(values => {
+        const number = String(get(values, 'number', '')).trim()
+        const accountNumber = String(get(values, 'accountNumber', '')).trim()
```

Comment:
Why not to implement this trimming in `resolveInput` hook of `Meter` schema?

Reply:
Yes, it happens in `resolveInput` hook

Reply:
removed it from submit handler